detox/runners/migration.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
```

Comment:
@noomorph this is quite thoughtful of you 😄
Are we going to keep this for good or could we get rid of that at some point?

Reply:
Detox 21 probably

Reply:
OK: https://github.com/wix/Detox/discussions/3406